README.md

```diff
@@ -145,11 +145,11 @@ To run the application from the CLI, run the following command from the root of
 
 ```bash
 cd LeaderboardBackend
-dotnet run  # or `dotnet watch` to run with hot reload 
+dotnet run  # or `dotnet watch` to run with hot reload
 ```
 
 #### Test the App
-Docker is required to run integration tests against a real Postgres database.  
```

Comment:
the 2 spaces at the end of the line should stay, that's how you do a line return in markdown

Reply:
Editor strips whitespace on save :upside_down_face: